azure-pipelines.yml

```diff
@@ -0,0 +1,96 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
```

Comment:
Do we need license in yaml files?

Reply:
Looks to be a bit of self publication from microsoft - it was an autogenerated document